Tests/PostgresNIOTests/New/Extensions/PSQLBackendMessage+Equatable.swift

```diff
@@ -1,4 +1,5 @@
 @testable import PostgresNIO
+import class Foundation.JSONEncoder
```

Comment:
Did Xcode annoyingly insert an unneeded `import` you didn't notice as it constantly does to me?